packages/wdio-logger/src/node.ts

```diff
@@ -26,37 +26,37 @@ const SERIALIZERS = [{
     /**
      * display error stack
      */
-    matches: (err) => err instanceof Error,
-    serialize: (err) => err.stack
+    matches: (err: any) => err instanceof Error,
```

Comment:
```suggestion
    matches: (err: Error) => err instanceof Error,
```

Reply:
Not sure if we can do this but I feel like it might be possible to assign the Error type here